datahub-web-react/src/app/lineage/utils/layoutTree.ts

```diff
@@ -292,7 +295,10 @@ function layoutColumnTree(
 
                 const sourceFieldY = currentNode?.y || 0 + 1;
                 let sourceFieldX = (currentNode?.x || 0) + 30 + sourceTitleHeight;
-                if (!collapsedColumnsNodes[currentNode?.data.urn || 'no-op']) {
+                // if currentNode is a dataJob, draw line from center of data job
+                if (currentNode?.data.type === EntityType.DataJob) {
+                    sourceFieldX = currentNode?.x || 0;
```

Comment:
Twice, once for source and another for destination case?

Reply:
yup exactly. when drawing that line in and out of datajobs, we need to make sure that the column arrow goes through the center of the data job instead of below where columns would be shown for other entities.

Reply:
got it